x-pack/plugins/security_solution/public/timelines/components/open_timeline/open_timeline.tsx

```diff
@@ -257,8 +257,8 @@ export const OpenTimeline = React.memo<OpenTimelineProps>(
                       <>
                         <UtilityBarText data-test-subj="selected-count">
                           {timelineType === TimelineType.template
-                            ? i18n.SELECTED_TEMPLATES(selectedItems.length)
-                            : i18n.SELECTED_TIMELINES(selectedItems.length)}
+                            ? i18n.SELECTED_TEMPLATES(selectedItems?.length || 0)
+                            : i18n.SELECTED_TIMELINES(selectedItems?.length || 0)}
```

Comment:
`Nit` : might be more clear to simply assign a default value of `[]` to `selectedItems` in prop destructuring. But I will leave it upto you.

Reply:
absolutely, fixed!